WordPress/Classes/ApplicationToken/ApplicationTokenItemView.swift

```diff
@@ -0,0 +1,85 @@
+import SwiftUI
+import DesignSystem
+
+struct ApplicationTokenItemView: View {
+    @State private var isConfirmingDeletion: Bool = false
+
+    private let token: ApplicationTokenItem
+
+    init(token: ApplicationTokenItem) {
+        self.token = token
+    }
+
+    var body: some View {
+        Form {
+            Section {
+                VStack(alignment: .leading) {
+                    Text(Self.name)
+                        .font(.headline)
```

Comment:
Let's switch a more conventional typography. A secondary bold font looks a bit off.

```suggestion
                VStack(alignment: .leading, spacing: 4) {
                    Text(Self.name)
                        .font(.subheadline)
```

<img width="320" alt="Screenshot 2024-08-19 at 9 16 10 AM" src="https://github.com/user-attachments/assets/a037ac2a-6a18-46a9-9592-5490d74324f5">

Reply:
The same change should apply to `DSListDetailItem`.